point/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+from user.models import User
+
+class Point(models.Model):
+    new_client = models.ForeignKey(User, on_delete=models.CASCADE, related_name='new')
+    old_client = models.ForeignKey(User, on_delete=models.CASCADE, related_name='old')
+    date       = models.DateTimeField()
```

Comment:
point 테이블에 point는 없나요??

```suggestion
class Point(models.Model):
    new_client = models.ForeignKey(User, on_delete=models.CASCADE, related_name='new')
    old_client = models.ForeignKey(User, on_delete=models.CASCADE, related_name='old')
    date       = models.DateTimeField()
```

Reply:
포인트가 회원가입을 했을 때 5000원 **고정**이고 이걸 가지고 구매에 사용하거나 하는 기능을 구현하지는 못할 것 같습니다. 그래서  별도로 추천으로 적립되는 포인트 금액을 기록할 필드는 넣지 않았습니다! 넣는 게 좋을까요??